content/recipes/introduction/public-kg-resource-integration.md

```diff
@@ -311,16 +285,14 @@ dl = OBL2021Dataset()
 train = dl.training # torch.tensor of shape(num_train,3)
 valid = dl.validation # torch.tensor of shape(num_val,3)
 ```
```

Comment:
@proccaserra can you write 1-2 sentences here explaining what the code does?

Reply:
will do